README.md

```diff
@@ -10,11 +10,11 @@ It works for both iOS and Android.
 
 ### Prerequisites
 
-To use LiveChat in your React application, you will need LiveChat license ID. 
+To use LiveChat in your React application, you will need the LiveChat license ID. 
 
-If you already have a LiveChat account, get your **license_id** [here](https://my.livechatinc.com/settings/code).
+To obtain authorization data (redirect_uri and client_id), you need to create a new LiveChat app. See our [Creating LiveChat apps](https://developers.livechatinc.com/docs/getting-started/guides/#creating-livechat-apps) documentation. 
```

Comment:
can we be more verbose about what is `client_id` and `redirect_uri`? I believe this is not mentioned in the linked article (also not available in the video linked there).